crates/driver/src/infra/liquidity/config.rs

```diff
@@ -137,6 +143,8 @@ impl UniswapV3 {
         Some(Self {
             router: deployment_address(contracts::UniswapV3SwapRouter::raw_contract(), network)?,
             max_pools_to_initialize: 100,
+            graph_api_base_url: Url::parse(&DEFAULT_GRAPH_API_BASE_URL)
+                .expect("invalid default Graph API base URL"),
```

Comment:
Shouldn't the url be configurable here as well?

Reply:
done